.mvn/wrapper/maven-wrapper.properties

```diff
@@ -5,14 +5,14 @@
 # to you under the Apache License, Version 2.0 (the
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
-# 
-#   http://www.apache.org/licenses/LICENSE-2.0
-# 
+#
+#   https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
this URL should not be changed - some code scanners expect the http form of the URL

this URL redirects to https anyway

Reply:
Such code scanners aren't doing anyone a favor. How's http better than https? 

Also, I didn't change this myself. This pull request was automatically generated